ColorzCore/Program.cs

```diff
@@ -8,6 +9,10 @@ namespace ColorzCore
     class Program
     {
         public static bool Debug = false;
+
+
+        public static ExecTimer Timer = null;

```

Comment:
Instead of a static variable, use the Singleton pattern on ExecTimer. 
nulls bad.

Reply:
~~i'm bad and don't understand what a singleton is in this context, but did my last commit fix this~~

Reply:
Lemme pull it and take care of it

Reply:
https://www.oodesign.com/singleton-pattern.html
Here's a link for reference. It's an Object Oriented design pattern, and I think this page talks about its motivation pretty well (and you should be able to see why it applies here).
(Note: I only got this page off Googling, no endorsement or anything in particular meant)